themes/aframe/layout/partials/secondary/sidebar_header.ejs

```diff
@@ -71,4 +71,14 @@ links = links.map(function (link) {
       </li>
     <% }) %>
   </ul>
+
+  <span class="sponsor" itemscope itemprop="organization" itemtype="http://schema.org/Organization">
```

Comment:
we can add this to a separate `sponsor.ejs` partial file (I can do it in a follow-up pull request; no need to worry about it)